fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
Part of what's confusing is that we are trying to reproduce representations, but we are trying to do that by storing resources. So one way to cut this knot would be to _merge_ all of the descriptions to disk and then accept that while we can give you back all of your information, we cannot give you back the exact behavior of the API when you exported. That's not what we want, though, I don't think.

So the principle I kind of want to take forward is to make the serialization look as much as possible like the API, which is not a profound thought, but in this case it gives us practical guidance: descriptions should be stored _as separate resources_. That would point to @escowles choice: use the request URI.